gridstatus/ercot.py

```diff
@@ -469,6 +787,42 @@ def _get_document(
         url = f"https://www.ercot.com/misdownload/servlets/mirDownload?doclookupId={doc[1]}"
         return url, doc[0]
 
+    def _get_documents(
```

Comment:
I thought about this a bit, and they're distinct since `_get_document` is opinionated about picking the latest AND raising an exception if there's no match. Would you want to preserve that behavior, even if it reuses `_get_documents`?

Reply:
i think the approach to preserve that behavior while reusing `_get_documents` is good